src/OVAL/oval_variable.c

```diff
@@ -340,6 +465,8 @@ struct oval_variable *oval_variable_new(struct oval_definition_model *model, con
 				return NULL;
 
 			evar = (oval_variable_EXTERNAL_t *) variable;
+			evar->possible_values = NULL;
```

Comment:
Is it worth the trouble to keep track of whether this is or isn't NULL?

Would we be better off with always allocating a collection here?